src/main/java/org/kohsuke/github/GHDiscussion.java

```diff
@@ -0,0 +1,58 @@
+package org.kohsuke.github;
+
+import java.io.IOException;
+import java.net.URL;
+
+/**
+ * A discussion in GitHub Team.
+ *
+ * @author Charles Moulliard
+ */
+public class GHDiscussion extends GHObject {
+
+    protected GitHub root;
+    protected GHOrganization organization;
+    protected GHTeam team;
+    protected String body, title, html_url;
```

Comment:
```suggestion
    @JacksonInject
    private GitHub root;
    private GHOrganization organization;
    private GHTeam team;
    private String body, title, html_url;
```

Reply:
Also, https://developer.github.com/v3/teams/discussions/#get-a-discussion shows a number of other fields (team_url, comments_count, etc).  Do you want to leave those out for now or add them?

Reply:
> Also, https://developer.github.com/v3/teams/discussions/#get-a-discussion shows a number of other fields (team_url, comments_count, etc). Do you want to leave those out for now or add them?

I suggest that we add them in a next PR and include them within the next release `1.114`. Make sense ?